content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- gsoc
```

Comment:
Just my curiosity asking, is there a plan in the future to filter blog posts by their tags? currently I don't think there is such a feature, right?